ephios/core/pdf.py

```diff
@@ -79,32 +148,12 @@ def get_story(self):
         table.setStyle([("VALIGN", (0, -1), (-1, -1), "TOP")])
         story.append(table)
         story.append(Spacer(height=0.5 * cm, width=15 * cm))
-
-        if participation_info := shift.structure.get_participation_display():
-            story.append(Paragraph(_("Participants"), self.style["Heading2"]))
-            col_count = len(participation_info[0])
-            table = Table(
-                [
-                    [[Paragraph(entry)] for entry in participation]
-                    for participation in participation_info
-                ],
-                hAlign="LEFT",
-                colWidths=[125 * mm / col_count] * col_count,
-            )
-            table.setStyle(
-                TableStyle(
-                    [
-                        ("INNERGRID", (0, 0), (-1, -1), 0.25, colors.black),
-                        ("BOX", (0, 0), (-1, -1), 0.25, colors.black),
-                    ]
-                )
-            )
-            story.append(table)
-
+        story.append(Paragraph(_("Participants"), self.style["Heading2"]))
+        story.append(self.get_shift_structure_data_table(shift))
         return story
 
 
-class MultipleShiftEventExporter(BasePDFExporter):
+class MultipleShiftEventExporter(EventExport):
     def __init__(self, event):
         self.event = event
         super().__init__(title=event.title)
```

Comment:
this can probably be reduced even further to have less code duplication (esp. thinking about the meeting time etc. for every shift that is defined here as well as in the single shift exporter

Reply:
You're right, I'd defer more modifications to a seperate PR with more regards to #1130